systemvm/patches/debian/config/etc/init.d/cloud-early-config

```diff
@@ -1207,33 +1145,27 @@ setup_secstorage() {
   fi
   setup_apache2 $ETH2_IP
 
+  # Deprecated, should move to Cs Python all of it
+  sed -e "s/<VirtualHost .*:80>/<VirtualHost $ETH2_IP:80>/" \
+    -e "s/<VirtualHost .*:443>/<VirtualHost $ETH2_IP:443>/" \
+    -e "s/Listen .*:80/Listen $ETH2_IP:80/g" \
+    -e "s/Listen .*:443/Listen $ETH2_IP:443/g" \
+    -e "s/NameVirtualHost .*:80/NameVirtualHost $ETH2_IP:80/g" /etc/apache2/vhost.template > /etc/apache2/sites-enabled/vhost-${ETH2_IP}.conf
```

Comment:
The NameVirtualHost search and replace seems to have failed.

I checked the contents of vhost-109.72.XX.XXX.conf on my test env and saw:

<pre>Listen 109.72.XX.XX:80
NameVirtualHost 10.1.1.1:80</pre>

The **sed** seems OK to me, ran it by hand and it worked as supposed.